packages/server/src/index.ts

```diff
@@ -1531,6 +1588,27 @@ export class App {
                 if (!isKeyValidated) return res.status(401).send('Unauthorized')
             }
 
+            if (incomingInput.uploads) {
+                // @ts-ignore
+                ;(incomingInput.uploads as any[]).forEach((upload: any) => {
+                    if (upload.type === 'file') {
+                        const filename = upload.name
+                        const dir = path.join(getUserHome(), '.flowise', 'gptvision', chatId)
+                        if (!fs.existsSync(dir)) {
+                            fs.mkdirSync(dir, { recursive: true })
+                        }
+                        const filePath = path.join(dir, filename)
+                        const splitDataURI = upload.data.split(',')
+                        const bf = Buffer.from(splitDataURI.pop() || '', 'base64')
+                        //TODO: check if file exists, what should we do if it exists?
```

Comment:
if file exists, we should replace it

Reply:
Resolved! that is the default behavior. updated comment to reflect the same